example/50-seed.yaml

```diff
@@ -25,7 +25,29 @@ spec:
       name: backup-secret
       namespace: garden
   dns:
-    ingressDomain: dev.my-seed.example.com
+    ingressDomain: ingress.dev.my-seed.example.com
+#   provider:
+#     type: aws-route53
+#     secretRef:
+#       name: ingress-secret
+#       namespace: garden
+#     domains:
+#       include:
+#       - my-custom-domain.com
+#       - my-other-custom-domain.com
+#       exclude:
+#       - yet-another-custom-domain.com
+#     zones:
+#       include:
+#       - zone-id-1
+#       exclude:
+#       - zone-id-2
+# ingress: # If set then `.spec.dns.ingressDomain` must be nil and `.spec.dns.provider` must be set.
+#   domain: ingress.dev.my-seed.example.com
```

Comment:
Do you think the API benefits from this new `domain` field? I tend to say that this makes the code and usability slightly confusing since now both fields are part of the API and we won't/can't remove them in the near future. Additionally, `ingressDomain` logically still fits under `dns`, so FMPOV no need to introduce this new field. WDYT?

Reply:
As discussed, I'm resolving this item and we stick to both fields because `ingressDomain` is only configurable if a non-managed Ingress controller is used, and `ingress.domain` if the manage Ingress controller is used i.e., mutual exclusion.